### ISSUE TITLE
Fix S1075 FN: Hardcoded path in object initialization are not reported

### ISSUE THREAD

**Issue by sebastien-marichal** [SonarSource/sonar-dotnet#7815]
```cs
var myClass = new MyClass
{
    FilePath = "c:\\blah\\blah\\blah.txt" // FN
};
```

S1075 should check for any assignation to a property in object initialization.
